src/components/CvAccordion/consts.js

```diff
@@ -1,16 +1,16 @@
 export const alignConsts = [null, 'start', 'end', ''];
 alignConsts.$labels = {
-  'Not supplied': 0,
-  Start: 1,
-  End: 2,
-  'Empty string': 3,
+  'Not supplied (end)': 0,
```

Comment:
I wonder if it would be more understandable to replace both "empty string" and "null" options with just single "default" or to be more precise "default (start)"? 
For me the meaning of (especially) "not supplied" was not fully clear

Reply:
![image](https://github.com/carbon-design-system/carbon-components-vue/assets/7536103/01e9750b-2a42-49c9-a603-41d99afe751e)

![image](https://github.com/carbon-design-system/carbon-components-vue/assets/7536103/10413e75-5f13-4a45-b248-8ce938bab49c)

Reply:
Yes that was confusing. Love the suggestion.